core/game_events.go

```diff
@@ -29,8 +30,12 @@ type PlayerDied struct {
 }
 
 type FoodEaten struct {
-	ID int     // ID of player who ate food
-	Pos Coord  // coordinates of eaten food
+	ID int		// eaten food ID
```

Comment:
Better to rename to `FoodID` and remove the comment (since one day, there also might be a `SnakeID`).